core_impl/src/test/java/io/opencensus/stats/ViewManagerImplTest.java

```diff
@@ -71,391 +92,394 @@
+    statsRecorder.record(
+        createContext(factory, KEY, VALUE),
+        MeasureMap.builder().set(MEASURE, 5.0).build());
+    clock.setTime(Timestamp.create(3, 3));
+    ViewData viewData1 = viewManager.getView(VIEW_NAME);
+    clock.setTime(Timestamp.create(4, 4));
+    ViewData viewData2 = viewManager.getView(VIEW_NAME_2);
+    assertThat(viewData1.getWindowData()).isEqualTo(
+        CumulativeData.create(Timestamp.create(1, 1), Timestamp.create(3, 3)));
+    StatsTestUtil.assertAggregationMapEquals(
+        viewData1.getAggregationMap(),
+        ImmutableMap.of(
+            Arrays.asList(VALUE),
+            StatsTestUtil.createAggregationData(AGGREGATIONS, 5.0)),
+        EPSILON);
+    assertThat(viewData2.getWindowData()).isEqualTo(
+        CumulativeData.create(Timestamp.create(2, 2), Timestamp.create(4, 4)));
+    StatsTestUtil.assertAggregationMapEquals(
+        viewData2.getAggregationMap(),
+        ImmutableMap.of(
+            Arrays.asList(VALUE),
+            StatsTestUtil.createAggregationData(AGGREGATIONS, 5.0)),
+        EPSILON);
+  }
+
+  @Test
+  public void testMultipleViewsDifferentMeasures() {
+    MeasureDouble measure1 =
+        Measure.MeasureDouble.create(MEASURE_NAME, MEASURE_DESCRIPTION, MEASURE_UNIT);
+    MeasureDouble measure2 =
+        Measure.MeasureDouble.create(MEASURE_NAME_2, MEASURE_DESCRIPTION, MEASURE_UNIT);
+    View view1 =
+        createDistributionView(
+            VIEW_NAME, measure1, AGGREGATIONS, Arrays.asList(KEY));
+    View view2 =
+        createDistributionView(
+            VIEW_NAME_2, measure2, AGGREGATIONS, Arrays.asList(KEY));
+    clock.setTime(Timestamp.create(1, 0));
+    viewManager.registerView(view1);
+    clock.setTime(Timestamp.create(2, 0));
+    viewManager.registerView(view2);
+    statsRecorder.record(
+        createContext(factory, KEY, VALUE),
+        MeasureMap.builder().set(measure1, 1.1).set(measure2, 2.2).build());
+    clock.setTime(Timestamp.create(3, 0));
+    ViewData viewData1 = viewManager.getView(VIEW_NAME);
+    clock.setTime(Timestamp.create(4, 0));
+    ViewData viewData2 = viewManager.getView(VIEW_NAME_2);
+    assertThat(viewData1.getWindowData()).isEqualTo(
+        CumulativeData.create(Timestamp.create(1, 0), Timestamp.create(3, 0)));
+    StatsTestUtil.assertAggregationMapEquals(
+        viewData1.getAggregationMap(),
+        ImmutableMap.of(
+            Arrays.asList(VALUE),
+            StatsTestUtil.createAggregationData(AGGREGATIONS, 1.1)),
+        EPSILON);
+    assertThat(viewData2.getWindowData()).isEqualTo(
+        CumulativeData.create(Timestamp.create(2, 0), Timestamp.create(4, 0)));
+    StatsTestUtil.assertAggregationMapEquals(
+        viewData2.getAggregationMap(),
+        ImmutableMap.of(
+            Arrays.asList(VALUE),
+            StatsTestUtil.createAggregationData(AGGREGATIONS, 2.2)),
+        EPSILON);
+  }
+
+  @Test
+  public void testGetDistributionViewDataWithoutBucketBoundaries() {
```

Comment:
This name is fairly specific to the old views (i.e. Distributions), consider renamining.